index.html

```diff
@@ -5,7 +5,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Części Samochodowe - wyszukiwarka codeVIN</title>
     <link rel="stylesheet" type="text/css" href="reset.css">
-    <link href="https://fonts.googleapis.com/css?family=Faster+One|Open+Sans" rel="stylesheet">
```

Comment:
You are deleting our extended font link :/ Please leave this line in code. It is very important for our page to correctly beeing displayed. Thanks :)